### PR TITLE
NextMillennium Bid Adapter: sends not one size, but multiple for a request

### PR DESCRIPTION
**Pull request by mhlm**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change

A small change that sends not one size, but several for a request to our server

### CODE REVIEW

**Comment by mhlm** [comment]
@patmmccann 1. Fixed tests for new changes
2. Added an update from the master prebid/Prebid.js branch

File: modules/nextMillenniumBidAdapter.js [hunk 1]
```diff
@@ -176,14 +176,14 @@ function getPlacementId(bid) {
   if (!groupId) return placementId
 
   let windowTop = getTopWindow(window)
-  let size = []
+  let sizes = []
   if (bid.mediaTypes) {
-    if (bid.mediaTypes.banner) size = bid.mediaTypes.banner.sizes && bid.mediaTypes.banner.sizes[0]
-    if (bid.mediaTypes.video) size = bid.mediaTypes.video.playerSize
+    if (bid.mediaTypes.banner) sizes = bid.mediaTypes.banner.sizes
```
Comment (patmmccann) [inline modules/nextMillenniumBidAdapter.js:181]: our style guide doesn't like all these implied semicolons

**Comment by mhlm** [comment]
@patmmccann I corrected your remark with a semicolon

**Review by patmmccann — APPROVED** [review]

**Comment by mhlm** [comment]
@jsnellbaker Good afternoon. Could you check our pull request?

**Review by jsnellbaker — APPROVED** [review]

**Comment by mhlm** [comment]
@jsnellbaker Thanks.

**Pull request merged**